.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+# Pull Request Checklist
+
+* [ ] Have you read [How to write the perfect pull request](https://github.com/blog/1943-how-to-write-the-perfect-pull-request)?
+* [ ] Have you read through the [contributor guidelines](https://github.com/otwcode/otwarchive/blob/master/CONTRIBUTING.md)?
+* [ ] Have you added tests for any changed functionality?
+* [ ] Have you added the JIRA issue number as the *first* thing in your pull request title (eg: `AO3-1234 Fix thing`)
+* [ ] Have you updated the JIRA issue with the information below?
+
+# Helpful things
+
+## Fixes
```

Comment:
Could we get rid of the word and heading fixes and keep it so the URL is the very first thing in the PR message?